go.mod

```diff
@@ -24,4 +24,4 @@ require (
 	google.golang.org/api v0.22.0
 )
 
-go 1.13
+go 1.14
```

Comment:
This line doesn't prevent recent Go 1.13 compilers from building the plugin. It just says "use the latest modules behavior." Keeping this up-to-date can prevent some dependency headaches.